lib/Benchmark.pm

```diff
@@ -494,7 +494,7 @@ init();
 
 sub BEGIN {
     if (eval 'require Time::HiRes') {
-	Time::HiRes->import('time');
+	 Time::HiRes->import('time');
```

Comment:
This now has an extraneous space.

Reply:
I see, I fixed that as well, though the inconsistent indentation is on the file-level.